fault_set.go

```diff
@@ -0,0 +1,88 @@
+// Copyright © 2019 - 2022 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
If it is a new file that we are introducing in 2023 then 
// Copyright © 2019 - 2022 Dell Inc. => // Copyright © 2023 Dell Inc.
What do you think about it?
This is applicable for all files. If it was created in 2019 and we are modifying it in 2023 then // Copyright © 2019 - 2023.

Reply:
Makes sense. Thanks

Reply:
Addressed. Please check